src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -145,6 +149,8 @@ static void print_help_messages() {
 debug_configuration::debug_configuration()
         : help(0)
         , verbose(0)
+        , verbose_color(0)
```

Comment:
How about set this as true by default?

Reply:
As you know, we can use it by environment, so color setting can be default by personal environment.
I think non color as default is better in personal.

Reply:
Hmm I am not sure "default non color is better" (I prefer colored default but yeah its nit...)